diff --git a/imam_datasub/lib/features/admin/presentation/screens/admin_dashboard_screen.dart b/imam_datasub/lib/features/admin/presentation/screens/admin_dashboard_screen.dart
index 3a24bdb..df540f0 100644
--- a/imam_datasub/lib/features/admin/presentation/screens/admin_dashboard_screen.dart
+++ b/imam_datasub/lib/features/admin/presentation/screens/admin_dashboard_screen.dart
@@ -1,5 +1,6 @@
 import 'package:flutter/material.dart';
 import 'package:flutter_riverpod/flutter_riverpod.dart';
+import 'package:go_router/go_router.dart';
 import '../../../../core/constants/app_colors.dart';
 import '../../../../core/constants/app_dimensions.dart';
 import '../../../../core/router/route_names.dart';
